instrumentation/spring/spring-boot-actuator-autoconfigure-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/actuator/v2_0/OpenTelemetryMeterRegistryAutoConfiguration.java

```diff
@@ -43,7 +43,7 @@ public MeterRegistry otelMeterRegistry() {
 
   @Bean
   // static to avoid "is not eligible for getting processed by all BeanPostProcessors" warning
-  static BeanPostProcessor postProcessCompositeMeterRegistry() {
+  public static BeanPostProcessor postProcessCompositeMeterRegistry() {
```

Comment:
```suggestion
  // public to avoid failure when using invoke dynamic
  // (https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13294)
  public static BeanPostProcessor postProcessCompositeMeterRegistry() {
```

Reply:
I used
```java
  // must be public because this class is injected as a proxy when using non-inlined advice and that
  // proxy contains only public methods
```